helm/alfresco-content-services/requirements.yaml

```diff
@@ -10,8 +10,8 @@ dependencies:
   repository: https://kubernetes-charts.storage.googleapis.com/
   condition: postgresql.enabled
 - name: alfresco-search
-  version: 0.0.11
-  repository: https://kubernetes-charts.alfresco.com/stable
+  version: 0.0.12
+  repository: https://kubernetes-charts.alfresco.com/incubator
```

Comment:
Not sure if we should pick up from incubator. There can come problem because during the moving from incubator to stable the version is notexisting in incubator anymore because it is a real move and not a copy paste. I would like to insist that we stay on stable

Reply:
That just happened to me in the past that a version wasn't in incubator anymore. As well I think it makes sense to only pick up from stable. For development reason you can override that locally :)

Reply:
I need to ask for a stable release of 0.0.12 as there are some changes there to use nameOverride